packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -81,6 +86,28 @@ class ExpansionPanel {
   /// The widget builder that builds the expansion panels' header.
   final ExpansionPanelHeaderBuilder headerBuilder;
 
+  /// The widget builder that builds the expansion panels' indicator.
+  ///
+  /// If null, [ExpandIcon] will be used as the expansion icon by default.
+  ///
+  /// {@tool sample}
+  ///
+  /// ```dart
+  /// ExpansionPanel(
+  ///  isExpanded: true,
+  ///  headerBuilder: (context, isExpanded) =>
```

Comment:
Arrow functions should only be used if the expression fits on one line
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods

Reply:
OK, I'll read the doc [https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods](url) and re-indentate my code

Reply:
It will cost me some time to learn how to write tests, so maybe tomorrow I will push another commit with reorganized codes and test codes.